app/__init__.py

```diff
@@ -32,20 +36,24 @@ def create_app(environment="development"):
 
     # Set up extensions.
     db.init_app(app)
-    login_manager.init_app(app)
+
+    bootstrap.init_app(app)
+    basic_auth.init_app(app)
+
+    # login_manager.init_app(app)
```

Comment:
no need login-manager in this project? So you simply remove it, no need to comment unused code